src/platform/testing/e2e/helpers.js

```diff
@@ -75,6 +77,14 @@ function disableAnnouncements(client) {
   });
 }
 
+function nwUploadTestFile(client, fileData) {
```

Comment:
I'm a little mixed on whether this should live here or in the HCA-specific helpers. But I think I do prefer to have it here. 👍

Reply:
I would prefer to call it simply `uploadTestFile` because I don't think that `nw` is universally known to mean `nightwatch`. At least it took me a little effort to override my instinct to think "northwest" :-p

Reply:
Now renamed to uploadTestfile.